modules/theme-tailwind-magento/magento/cms/blocks/pb-products.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <ProductCarousel v-if="products.length" :products="products" />
+  <ProductCarousel v-if="products?.length" :products="products" />
```

Comment:
Why? If the products might be missing, there should be default empty array in the usePbProducts composable, no here in the template.